src/pages/Home.jsx

```diff
@@ -1,6 +1,136 @@
+import placeholderImage from '/src/assets/placeholder-image.jpg';
```

Comment:
reference this to make the home page 

https://www.figma.com/design/EqVOg6i2403VobrJvadONm/FUSION-Website-Design-File?node-id=1122-1462&t=gN9HYhSRVefFOCum-4